gapis/api/cmd_id_group.go

```diff
@@ -311,6 +351,72 @@ func (g *CmdIDGroup) AddGroup(start, end CmdID, name string) error {
 	return nil
 }
 
+// AddRoot adds a new Subcommand Root for the given index.
+// It returns the span for this SubcommandGroup
+func (g *CmdIDGroup) AddRoot(rootidx []uint64) *Span {
```

Comment:
It is unusual and usually broken to return a pointer to an interface. Maybe return `*SubCmdRoot`?